src/udev/udev-event.c

```diff
@@ -634,6 +640,9 @@ static int on_spawn_sigchld(sd_event_source *s, const siginfo_t *si, void *userd
 
 static int spawn_wait(Spawn *spawn) {
         _cleanup_(sd_event_unrefp) sd_event *e = NULL;
+        _cleanup_(sd_event_source_unrefp) sd_event_source *sigchld_source = NULL;
+        _cleanup_(sd_event_source_unrefp) sd_event_source *stdout_source = NULL;
+        _cleanup_(sd_event_source_unrefp) sd_event_source *stderr_source = NULL;
```

Comment:
i'd probably define these in one line, but doesn't matter

Reply:
If these were on one line does the _cleanup_ attribute have to be duplicated for each pointer?
If you wish to change them to one line feel free to edit it once I update PR